src/storage/src/table/state_table.rs

```diff
@@ -266,8 +269,38 @@ impl<S: StateStore> StateTable<S> {
     }
 }
 
-pub type RowStream<'a, S: StateStore> = impl Stream<Item = StorageResult<Cow<'a, Row>>>;
+/// Iterator functions.
```

Comment:
Let's remove this and add a generic of `RowSerde` to the type alias `RowStream`.

```rust
pub type RowStream<'a, S: StateStore, RS: RowSerde> = impl Stream<Item = StorageResult<Cow<'a, Row>>>;
```